extensions/odoo_theme/static/style.scss

```diff
@@ -212,8 +208,6 @@ header.o_main_header {
     }
     @include media-breakpoint-up(xl) {
         width: $o-side-nav-width;
-        padding-top: $padding-l;
-        padding-left: $padding-l;
```

Comment:
By removing this, the main content is no longer aligned with the header. Line 405 needs adapting too.